datasets/_global/iadb_sanctions/crawler.py

```diff
@@ -74,7 +72,7 @@ def crawl(context: Context):
             prop = "nationality" if schema == "Person" else "jurisdiction"
             entity.add(prop, country)
 
-        end_dates = h.parse_date(row.pop("to", None), FORMATS)
+        end_dates = h.parse_date(row.pop("to", None), context.dataset.dates.formats)
```

Comment:
I think it's better to use `context.dataset.dates.formats` in this case , since it makes the establishment of ```
 if is_active:
            entity.add("topics", "debarment")
``` much more intuitive

Reply:
specifically, the date can be in the future, so we compare to today.

Reply:
in this case we have the entity at hand. Do we prefer

```python
h.apply_date(sanction, raw_end_date)
is_active = sanction.get("endDate") > [todays_date]
```
or
```
end_date = h.parse_date(raw_end_date, dataset.dates.formats)
is_active = end_date > [todays_date]
sanction.add("endDate")
```
The former feels a bit like we're shoe-horning into the helper, but it's more concise and has the benefit of more consistent date assignment across the codebase